architecture/evm/evm_state_poller.go

```diff
@@ -150,14 +150,24 @@ func (e *EvmStatePoller) Bootstrap(ctx context.Context) error {
 				subCtxErr := nctx.Err()
 				cancel()
 				if err != nil {
+					severity := common.ClassifySeverity(err)
 					if errors.Is(subCtxErr, context.DeadlineExceeded) {
-						e.logger.Error().Err(err).
-							Msgf("failed to poll evm state due to sub-context timeout after %f seconds", timeout.Seconds())
+						if severity == common.SeverityCritical {
+							e.logger.Error().Err(err).
+								Msgf("failed to poll evm state due to sub-context timeout after %f seconds", timeout.Seconds())
+						} else {
+							e.logger.Warn().Err(err).
+								Msgf("failed to poll evm state due to sub-context timeout after %f seconds", timeout.Seconds())
+						}
```

Comment:
Only use warn on a timeout:
```suggestion
					e.logger.Warn().Err(err).
						Msgf("failed to poll evm state due to sub-context timeout after %f seconds", timeout.Seconds())
```